org.adempiere.base/src/org/compiere/model/MOrder.java

```diff
@@ -2388,7 +2388,8 @@ protected MOrder createCounterDoc()
 		{
 			MOrderLine counterLine = counterLines[i];
 			counterLine.setOrder(counter);	//	copies header values (BP, etc.)
-			counterLine.setPrice();
+			//counterLine.setPrice();
```

Comment:
If we are sure that this is not needed, we should remove instead of just comments it out.